build/images/deps/nanoserver-version

```diff
@@ -1 +1 @@
-1809
+ltsc2022
```

Comment:
After adding support for Windows Server 2022 in Antrea, should we also announce that we will no longer maintain Antrea on Windows Server 2019 at the same time?

Reply:
I think so...

Reply:
@wenyingd I also created a PR to add base-image verification pipeline in CI, could you review it first? https://github.com/antrea-io/antrea/pull/6262